x/txreward/client/cli/tx.go

```diff
@@ -0,0 +1,53 @@
+package cli
+
+import (
+	"github.com/commercionetwork/commercionetwork/x/txreward/internal/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	"github.com/spf13/cobra"
+)
+
+func GetTxCmd(cdc *codec.Codec) *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "CommercioTXREWARDS transactions subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+	txCmd.AddCommand(GetCmdIncrementBlockRewardsPool(cdc))
+
+	return txCmd
+}
+
+func GetCmdIncrementBlockRewardsPool(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "increment-rewards-pool [coin-denom] [amount]",
```

Comment:
Can we rename this command to the simpler `deposit`? Also, I think we can simply use one parameter as the `send` command does in order to read the coin.

```suggestion
		Use:   "deposit [amount]",
```